bullet_train/app/models/concerns/memberships/base.rb

```diff
@@ -151,5 +155,9 @@ def remove_user_profile_photo
     user_profile_photo.purge
   end
 
+  def publish_changed_quantity
+    ActiveSupport::Notifications.instrument("quantity-changed", {team:})
```

Comment:
The name of this event seems too generic to me. I think we should do something to indicate _which_ quantity has changed. Maybe something like:

```suggestion
    ActiveSupport::Notifications.instrument("memberships.quantity-changed", {team:})
```

Reply:
Good point!